spyderplugins/widgets/profilergui.py

```diff
@@ -103,13 +103,13 @@ def __init__(self, parent, max_entries=100):
 
         self.collapse_button = create_toolbutton(self,
                                                  icon=ima.icon('collapse'),
-                                                 triggered=lambda dD=-1:
-                                                 self.datatree.change_view(dD),
+                                                 triggered=lambda dD:
+                                                 self.datatree.change_view(-1),

```

Comment:
Apparently the click handler is always called with the parameter `False`, which overrides the value of `dD`. So now the lambda parameter is nor read, so that the buttons have an effect.